modules/gateways/callback/blockonomics.php

```diff
@@ -45,7 +45,13 @@
 $invoiceId = $order['order_id'];
 $bits = $order['bits'];
 
-if($status == 0) {
+if($status != 2 && $status != 1 && $status != 0) {
```

Comment:
Good check, but not necessary can be removed

Reply:
Removed with commit 1232446